### PR TITLE
Add extended test for installonly packages

### PR DESCRIPTION
**Pull request by j-mracek**
The original test is ok, but this test is for a random behavior of the code, therefore it uses more additional packages in test to triger random behavior.

Required for: https://issues.redhat.com/browse/RHEL-15902

### CODE REVIEW

File: dnf-behave-tests/dnf/installonly.feature [hunk 1]
```diff
@@ -258,6 +258,58 @@ Scenario: Do not remove or change reason after remove of one of installonly pack
...
+        | install       | abcde-0:2.9.2-1.fc29.noarch           |
+        | install-dep   | wget-0:1.19.5-5.fc29.x86_64           |
+        | install-weak  | flac-0:1.3.2-8.fc29.x86_64            |
+   When I execute dnf with args "mark remove abcde"
+   Then the exit code is 0
+    And package reasons are
+        | Package                                | Reason          |
+        | abcde-2.9.2-1.fc29.noarch              | dependency      |
+        | flac-1.3.2-8.fc29.x86_64               | weak-dependency |
+        | wget-1.19.5-5.fc29.x86_64              | dependency      |
```
Comment (m-blaha) [inline dnf-behave-tests/dnf/installonly.feature:278]: For the sake of future maintainers, I'd like to include a comment here explaining why the `abcde` package is needed on the system, with a `dependency` reason. I'm concerned that without it, anyone attempting to understand the test in a year or two might become confused.
What do you think?

Comment (j-mracek) [reply dnf-behave-tests/dnf/installonly.feature:278]: Good point

Comment (j-mracek) [reply dnf-behave-tests/dnf/installonly.feature]: I have created a comment.

**Review by m-blaha — APPROVED** [review]
Thank you!

**Pull request merged**